src/views/dependencyDataProvider.ts

```diff
@@ -71,9 +71,9 @@ export class DependencyDataProvider implements TreeDataProvider<ExplorerNode> {
     public async revealPaths(paths: INodeData[]): Promise<DataNode> {
         const projectNodeData = paths.shift();
         const projects = await this.getRootProjects();
-        const correspondProject = <DataNode>projects.find((node: DataNode) =>
-            node.path === projectNodeData.path && node.nodeData.name === projectNodeData.name);
-        return correspondProject.revealPaths(paths);
+        const correspondProject = projects ? <DataNode>projects.find((node: DataNode) =>
```

Comment:
correspond is VI, please change it to ADJ.

Reply:
The correspond prefix is not necessary. Every variable is correspond variable in the code.